src/App.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
-import logo from './logo.svg';
-import Mockman from "mockman-js";
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import Header from "./components/Header";
+import { ModalProvider } from "./contexts/ModalContext";
+import { AuthProvider } from "./contexts/AuthDialogContext";
+import { Loader } from "./components/Loader";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+
 
 function App() {
```

Comment:
suggestion: import `FC` from "react" and make them the standard functional components in ts

Reply:
fixed. thanks for mentioning it